packages/stb_image/stb_image.0.4/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+maintainer: "Frederic Bour <frederic.bour@lakaban.net>"
+authors: "Frederic Bour <frederic.bour@lakaban.net>"
+homepage: "https://github.com/let-def/stb_image"
+bug-reports: "https://github.com/let-def/stb_image"
+license: "CC0"
+dev-repo: "https://github.com/let-def/stb_image.git"
+build: [make]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "stb_image"]
+depends: [
+  "ocamlfind" {build}
+  "result"
+]
+available: [ocaml-version != "4.01.0"]
```

Comment:
I'm curious, why is there such a constraint ?

Reply:
```
type 'kind buffer = ('a, 'b, c_layout) array1.t
  constraint 'kind = ('a, 'b) kind
```
doesn't typecheck with 4.01.0. Yet it works with 4.00 and >4.02.